flatbuffers/Message.fbs

```diff
@@ -42,7 +40,6 @@ union Body {
     ExecutionReport,
     Heartbeat,
     MarketDataRequest,
-    MarketDataAcknowledgement,
```

Comment:
This will break the union id's. We'll need to confirm how to do this, we may need to assign the specific ids.

Reply:
The union translates to:

```
export enum Body {
  NONE = 0,
  ExecutionReport = 1,
  Heartbeat = 2,
  MarketDataRequest = 3,
  MarketDataAcknowledgement = 4,
  MarketDataSnapshot = 5,
  NewOrderSingle = 6,
  OrderCancelReject = 7,
  OrderCancelRequest = 8,
  OrderReplaceRequest = 9,
  Reject = 10,
  TestRequest = 11,
  Quote = 12,
  QuoteAcknowledgement = 13,
  QuoteCancel = 14,
  QuoteRequest = 15,
  QuoteRequestReject = 16,
  MarketDataRequestReject = 17
}
```
so to remove we'll need to assign the ids directly in the union e.g.

```
union Body {
   ExecutionReport = 1,
   ...
```